#SAK-10413 revision assignment attachments are missing after importing a site using Site Archive tool 
#Cherry Pick
=svnm= https://source.sakaiproject.org/svn/assignment/trunk@47227 assignment

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-10413.tmp
@@ -0,0 +1 @@
+ 
